src/platform/forms-system/src/js/review/ObjectField.jsx

```diff
@@ -184,11 +184,13 @@ class ObjectField extends React.Component {
         {!formContext?.hideHeaderRow && (
           <div className="form-review-panel-page-header-row">
             {((titleString && title.trim()) || !titleString) &&
-              !formContext?.hideTitle && (
-                <h4 className="form-review-panel-page-header vads-u-font-size--h5">
-                  {title}
-                </h4>
-              )}
+            !formContext?.hideTitle ? (
```

Comment:
just a stylistic preference, but i would probably reverse this conditional so that it's a little more readable (i.e. removing the `!` and doing something like `formContext?.hideTitle ? <header> : <title>`) but this is fine as is!

Reply:
There is a line before the `formContext?.hideTitle` which has logic that I didn't want to mess with:

```js
{((titleString && title.trim()) || !titleString) &&
  !formContext?.hideTitle ? <h4 /> : <div />
```

Reply:
Sorry @anniebtran, this PR is going to need another review please :(